homeassistant/components/sensor/const.py

```diff
@@ -431,6 +431,7 @@ class SensorDeviceClass(StrEnum):
     - SI /metric: `m/s`, `km/h`
     - USCS / imperial: `ft/s`, `mph`
     - Nautical: `kn`
+    - Beaufort: `Bft`
```

Comment:
We should add this also to the docstring of `SPEED` above

Reply:
You're right

Reply:
Yes, so can you do that change please :)

Reply:
Sorry, only now I see what's missing